### DISCUSSION TITLE
Is there a Discord?

### DISCUSSION THREAD

**Discussion by tommy141x** [relagit/relagit#32] [General]
If there isn't a Discord I think having one would be a great way for people to communicate about this project 👍

**Comment by TheCommieAxolotl** [comment]
Great suggestion, In fact there is one however it hasn't been public up until now.

I'll post a link here and on the homepage/readme in a short time!

**Reply by TheCommieAxolotl** [reply]
@tommy141x Here you go: https://discord.gg/ptc9nku6tD
